projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -0,0 +1,1790 @@
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation[] =
+  [
+    {
+      class: 'QuickOrderFormComponent',
+      importPath: '@spartacus/cart/quick-order/components',
+      deprecatedNode: 'globalMessageService',
```

Comment:
`globalMessageService` is constructor parameter. We already handle it in "constructor" deprecation. Do we also process it in "methods and properties" deprecation?

Reply:
I think so, this schematics covers the use case of a class that extends QuickOrderFormComponent and uses the globalMessageService property in its code.